startupscript/butane/gcp/set-metadata.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
Should be a comment here about what the script does (creates a function) and that the expectation is that it is sourced.